sdk/Metrics/Exceptions/CantBeExported.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Sdk\Metrics\Exceptions;
+
+class CantBeExported extends ExportException
```

Comment:
Could we potentially just have `CantBeExported` extend `Exception` rather than the nested `CantBeExported -> ExportException -> Exception` tree?  Maybe I'm missing the reason you're doing it this way.

Maybe we could have multiple Exception classes defined in one `sdk/Metrics/Exceptions` file rather than in individual files.

Reply:
Well, we have 2 kind of exceptions: the ones where there is no reason to retry export (wrong data for example) and the ones where we can retry (the server is unreachable for now but we can try later).

The idea was to keep a single type of exceptions (`ExportException`) for all export operations so they can be easily catched and in the same time we could easily differ retryable ones from irretryable ones

Reply:
Cool sounds good to me